RELEASE.md

```diff
@@ -118,3 +118,16 @@ Generate the release tarballs using `contrib/scripts/release.sh` script:
 This will generate tarballs and associated checksum files in the `release`
 directory. Make sure to upload these tarball and checksum to the GitHub release
 page.
+
+## (OPTIONAL) Update `stable.txt` in the master branch
```

Comment:
just how optional?

Reply:
it's somewhat fuzzy right now. if you are releasing the latest version (say v0.6.2), you most likely want to update `stable.txt` so that people start picking up bug fixes and what not.

if you are releasing a patch version of older release (v0.5.2 for example), you don't want update `stable.txt` because it's already pointing to v0.6.1.

Reply:
Btw, this is good info that provides a lot more context than `optional`

Reply:
agreed. will update the section

Reply:
updated RELEASE.md